front/lib/models/assistant/conversation.ts

```diff
@@ -376,6 +377,10 @@ ContentFragment.init(
       type: DataTypes.TEXT,
       allowNull: false,
     },
+    url: {
```

Comment:
I thiiink we want to add a type as well. We can start with type "slack_thread_content" we'll add other content fragment types in the future this will avoid the switch on the slack.comm url which is not super future proof

Reply:
done